src/app/organization/bulk-import/view-bulk-import/view-bulk-import.component.ts

```
@@ -255,7 +255,21 @@ export class ViewBulkImportComponent implements OnInit {
       case BulkImportsConstants.CLIENT_TRANSFER_IMPORT:
         return !this.getSelectedOfficeId();
       case BulkImportsConstants.SAVINGS_TRANSACTIONS_IMPORT:
-        return !this.getEffectiveOfficeId();
+        const countryId = this.bulkImportForm.get('countryId')?.value;
```

Biome (2.4.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)